extensions/kafka/src/main/java/io/deephaven/kafka/KafkaPublishOptions.java

```diff
@@ -34,12 +39,20 @@ public static Builder builder() {
     public abstract Table table();
 
     /**
-     * The kafka topic to publish to.
+     * The default kafka topic to publish to.
      *
-     * @return the kafka topic
+     * @return the default kafka topic
      */
+    @Nullable
```

Comment:
Maybe you want to add a method that converts this with an `Optional.ofNullable()`?